hack/windows/Install-OVS.ps1

```diff
@@ -251,6 +255,13 @@ function ConfigOVS() {
     ovs-vsctl --no-wait set Open_vSwitch . ovs_version=$OVS_VERSION
 }
 
+if ($ContaineriseOVS -eq $true) {
```

Comment:
For ContaineriseOVS mode, I think only ovs kernel driver need to be installed?

Reply:
This is a different check , this is for checking whether containerd is the default runtime of the node on which we are containerising OVS... I think the check you are looking for is line 273

Reply:
I didn't find ContaineriseOVS in your updated script, please always run and verify your script after making updates.